Python/Product/Analysis/LanguageServer/Server.cs

```diff
@@ -126,7 +126,7 @@ private sealed class ReloadModulesQueueItem : IAnalyzable {
         #region Client message handling
         public override async Task<InitializeResult> Initialize(InitializeParams @params) {
             ThrowIfDisposed();
-            await DoInitializeAsync(@params);
+            await DoInitializeAsync(@params, CancellationToken);
```

Comment:
`CancellationToken` only has a value when it's used within a `using (AllowRequestCancellation)` block.